experimental/batchmap/sumdb/build/map.go

```diff
@@ -67,28 +67,24 @@ func main() {
 	flag.Parse()
 	beam.Init()
 
-	sumDB := filepath.Clean(*sumDB)
-	if sumDB == "" {
+	if *sumDB == "" {
```

Comment:
general nit: use len(string) rather than `... == ""` as it copes with `nil` correctly too.